### DISCUSSION TITLE
bash scripts seem to be unable to edit files when run with i3?? why isnt this working?

### DISCUSSION THREAD

**Discussion by AVDambeck** [i3/i3#6145] [Q&A] [answered]
Hi, I wrote some scripts to adjust Redshift, but they dont work when I run them from i3. https://pastebin.com/07wnEgUF     

the scripts work 100% when i run them from the terminal. when from i3, plus/minus do nothing; reset will do the redshift -x part, but will not change the value in the file 

seems like its having an issue writing to the files?? not sure whats up tbh. why isn't this working?

**Comment by okraits** [comment]
* Are the scripts executable?
* Do the keybindings work with another command which you know is working?
* Your scripts don't contain a shebang so a different shell might be used to execute the scripts than you might expect

**Comment by AVDambeck** [comment]
yes they are executable

yes the keybindings work with other commands

added a shebang to the scripts to no effect

**Reply by orestisfl** [reply, marked as answer]
From a quick look, your scripts use relative paths for `temp.var` and `increment.var` so when your run them from the same dir they work but when on the home dir, they won't.

Run `cd ~ && ~/programs/i3redshift/minus.sh` from the terminal and they shouldn't work.

**Reply by AVDambeck** [reply]
ahh this is it! didnt think to check them from a different dir. just finished writing them and they work now

my new scripts https://pastebin.com/1UTiqxbL
(still a little buggy, not sure why yet, but not an i3 problem)
